tests/providers/amazon/aws/hooks/test_glacier.py

```diff
@@ -0,0 +1,180 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import mock
+import pytest
+from botocore.exceptions import ClientError
+from testfixtures import LogCapture
+
+from airflow.providers.amazon.aws.hooks.glacier import GlacierHook
+
+VAULT_NAME = "airflow"
+JOB_ID = "1234abcd"
+REQUEST_RESULT = {"jobId": "1234abcd"}
+RESPONSE_BODY = {"body": "data"}
+JOB_STATUS = {"Action": "", "StatusCode": "Succeeded"}
+
+
+class TestAmazonGlacierHook(unittest.TestCase):
+    def setUp(self):
+        with mock.patch("airflow.providers.amazon.aws.hooks.glacier.GlacierHook.__init__", return_value=None):
+            self.hook = GlacierHook(aws_conn_id="aws_default")
+
+    @mock.patch("airflow.providers.amazon.aws.hooks.glacier.GlacierHook.get_conn")
```

Comment:
Should we mock this in `setUp` to do not repeat ourselves?

Reply:
Frankly speaking, I haven't seen such solution before.

Reply:
Examples:
https://github.com/apache/airflow/blob/ce19657ec685abff5871df80c8d47f8585eeed99/tests/providers/google/cloud/hooks/test_automl.py#L58-L59

But no strong opinion here, as long as the mocking works I'm fine

Reply:
Thanks :D